pysidt/sidt.py

```diff
@@ -9,10 +9,21 @@
 import json
 from sklearn import linear_model
 import scipy.sparse as sp
+import scipy
 
 logging.basicConfig(level=logging.INFO)
 
 
+class Rule:
```

Comment:
This class won't json serialize so it should break write_nodes and read_nodes as currently written.

Reply:
I added a test to make sure that `write_nodes` and `read_nodes` work with `Rule`